File: app/src/main/java/org/mifos/mobile/ui/views/SavingsAccountApplicationView.java
package org.mifos.mobile.ui.views;

/*
 * Created by saksham on 30/June/2018
 */

import org.mifos.mobile.models.templates.savings.SavingsAccountTemplate;
import org.mifos.mobile.ui.views.base.MVPView;

public interface SavingsAccountApplicationView extends MVPView {

    void showUserInterfaceSavingAccountApplication(SavingsAccountTemplate template);
    void showSavingsAccountApplicationSuccessfully();
    void showUserInterfaceSavingAccountUpdate(SavingsAccountTemplate template);
    void showSavingsAccountUpdateSuccessfully();
    void showError(String error);
    void showMessage(String showMessage);
}

File: app/src/main/java/org/mifos/mobile/ui/views/SavingsAccountApplicationView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.models.templates.savings.SavingsAccountTemplate
import org.mifos.mobile.ui.views.base.MVPView

/*
* Created by saksham on 30/June/2018
*/   interface SavingsAccountApplicationView : MVPView {
    fun showUserInterfaceSavingAccountApplication(template: SavingsAccountTemplate?)
    fun showSavingsAccountApplicationSuccessfully()
    fun showUserInterfaceSavingAccountUpdate(template: SavingsAccountTemplate?)
    fun showSavingsAccountUpdateSuccessfully()
    fun showError(error: String?)
    fun showMessage(showMessage: String?)
}